css/layout.css

```diff
@@ -53,7 +115,7 @@
     width: 80%;
   }
 
-  .col-5 > a > img {
-    max-height: 30rem;
+  .col-half {
+    width: 90%;
   }
 }
```

Comment:
Your media queries should all be `min-width`. Your base styles above your media queries should be targeting your styles on a phone, and then you should be using your media queries to adjust your styling as you increase in screen size